### ISSUE TITLE
[BUG_SP] AI worker end of up/down turns unusual backup behavior

### ISSUE THREAD

**Issue by kycoss** [Courseplay/Courseplay_FS22#592]
**Steps to reproduce**
Setup generated course with 0-2 headlands/ smooth corners and up/down field center, clockwise, 0 rows to skip, up/down angle automatic. 

Any wheeled tractor & 18m seeder or planter. Running 2 of them on 34m work width with one +8.5m offset hoz. and other -8.5m offset hor.  

**Actual behavior**
At the end of a field center run, tractor backs up extremely harsh and jackknifes against the planter. This wasn't an issue in v. 7.0.0.19 yesterday but updated to current 7.0.020 and issue has become prevalent. Settings are identical in game

**Expected behavior**
Tractor and planter to continue moving forward and rounding the turn, to end up in next portion of land

**Screenshots**
CP bad turn with 7.0.0.20 https://youtu.be/b0cmjrhxExA
CP good turn with 7.0.0.19 https://youtu.be/tZERHLTKvtQ

**Courseplay version**
7.0.019 & 7.0.020

**Log**
attached

**Map**
Deercreek mod map, occurred on elm creek as well

**Vehicles/ implements involved**
1050 fendt tractor  jd 8r 310 tractor and grain plains mod planter

[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/7892265/log.txt)

**Comment by Tensuko** [comment]
https://github.com/Courseplay/Courseplay_FS22/wiki/What-type-of-Turns-and-when-they-are-used.
And we need debug settings activated.
What "grain plains mod planter" is it ? ModHub ? Link ?

**Comment by kycoss** [comment]
https://farming-simulator.com/mod.php?lang=en&country=us&mod_id=226891&title=fs2022 modhub great plains seeder. Tomorrow ill go on (its 11pm here) and replicate with debug settings with the other long planters and seeders as it seems to be common among them and update

**Comment by pvaiko** [comment]
We really need those debug logs. Also, on the .20 video there is no offset and the working width seems to be the standard width.

**Comment by kycoss** [comment]
Here you go i did a few more tests & both with debug showing

https://youtu.be/YuxvBCUns6c is a amazon 15m seeder with offset that is a good turn and works well
https://youtu.be/qnvYZ-WaCjg is the great plains seeder with same offset for the size and it binds on the turn
I've randomly tried a few different implements and it seems to be just the mod hub one

**Comment by pvaiko** [comment]
The log file is what we really need, the log.txt file.

**Comment by kycoss** [comment]
[log.txt](https://github.com/Courseplay/Courseplay_FS22/files/7902330/log.txt)
Here you go

**Comment by pvaiko** [comment]
For the life of me, I can't reproduce this, at least not with the Great Plains. Also, in the log you now have the NZ Extreme, but you are saying it is only the Great Plains. Confusing.

So all we can do now is that you send a save game, and also, save the course you are using and posting that here too.

**Comment by kycoss** [comment]
So the log file issue was most likely my mistake as i quit of the game then decided to go back in and try other implements including the nz extreme to see if the issue occurred which it did not. 

I also went into my game save file without anything changing and tried the course again and it didn't bind up. I had saved everything immediately after i made the videos above so I'm unsure of what changed but it seems the pathing is correct now and i can't reproduce the issue myself. 

I would suggest closing this ticket, and I will keep playing around with debug on this time and if i notice the issue again i will just submit a new ticket with the saved game file right off the bat. 

Thanks for all the hard work your group is putting in for this mod.
